bot-brain/brain.js

```diff
@@ -0,0 +1,111 @@
+require('dotenv').config();
+let { Pinecone } = require('@pinecone-database/pinecone');
+let { OpenAIEmbeddings } = require("@langchain/openai");
+let { loadQAStuffChain } = require("langchain/chains");
+let { Document } = require("langchain/document");
```

Comment:
we can do it with openai package as well, since i was reading a blog of langchain and they have used like this.
we can leverage `openai.embeddings.create` in place of `OpenAIEmbeddings` and manually set context to avoid use of `loadQAStuffChain` and `Document`.